test/net/sourceforge/kolmafia/persistence/FaxBotDatabaseTest.java

```diff
@@ -91,21 +91,16 @@ public void coverageForInvalidMonster() {
 
     @Test
     public void exerciseSomeFaxbotOverridesForCoverage() {
-      FaxBotDatabase.FaxBot nullBot = null;
       FaxBotDatabase.FaxBot botA = new FaxBotDatabase.FaxBot("A Bot", 1999);
       FaxBotDatabase.FaxBot botAlsoA = new FaxBotDatabase.FaxBot("A Bot", 1999);
       FaxBotDatabase.FaxBot botB = new FaxBotDatabase.FaxBot("B Bot", 1999);
       FaxBotDatabase.FaxBot botInvalid = new FaxBotDatabase.FaxBot(null, 1999);
-      assertTrue(botA.equals(botA));
```

Comment:
this is valid contract testing -- `equals` should be reflexive.

You could implement `equals` such that this wasn't true, e.g. by implementing it as `return false`.